### PR TITLE
#151168136 effective onboarding

### PR DESCRIPTION
**Pull request by segunolalive**
#### What does this PR do?
gives a user a tour of the game interface

#### Description of Task to be completed?
Users should be quickly and effectively onboarded onto the app

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#151168136
#### Screenshots (if appropriate)
localhost-3000- iphone 6 plus 1
screen shot 2017-10-10 at 5 44 50 pm
screen shot 2017-10-10 at 5 44 56 pm

### CODE REVIEW

**Review by nebanat — APPROVED** [review]

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -8,11 +8,14 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
...
 script(src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js")
 script(src="/js/cfh.js")
+script(src="/lib/introjs/introjs.js")
 
 //AngularJS
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular.js')
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-resource.js')
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-cookies.js')
+script(type='text/javascript', src='https://cdnjs.cloudflare.com/ajax/libs/intro.js/2.7.0/intro.js')
+
```
Comment (ibonly) [inline app/views/includes/foot.jade]: Delete empty line, one empty line is ok.

File: app/views/includes/foot.jade [hunk 2]
```diff
@@ -49,6 +52,7 @@ script(type='text/javascript', src='/js/init.js')
 script(type='text/javascript', src='/js/controllers/dashboard.js')
 script(type='text/javascript', src='/js/controllers/password.js')
 
+
```
Comment (ibonly) [inline app/views/includes/foot.jade]: Remove empty line

File: public/views/tour.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  </header>
+  <div class="row">
+    <div class="container">
+      <div class="control-game">
+        <span id="tweet-container">
+          <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://cfh.io" data-text="Cards for Humanity: A Game for Horrible People Desperately Trying to do Good" data-related="CFH_App">Tweet</a>
+        <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
+        </span>
+        <button id="abandon-game-button" class="btn red darken-4 right" ng-click="abandonGame()">
+          Abandon Game
```
Comment (ibonly) [inline public/views/tour.html]: I thought this has been changed to `Leave Game`?

Comment (segunolalive) [reply public/views/tour.html]: Yes. It has. Actually, this file should have been deleted. No longer using this

**Review by ibonly — CHANGES_REQUESTED** [review]
Weldone, update review!!!

**Review by dinobi — APPROVED** [review]

**Review by ibonly — APPROVED** [review]

**Pull request merged**